paystack/src/main/java/co/paystack/android/api/model/ChargeResponse.kt

```diff
@@ -22,6 +23,22 @@ data class ChargeResponse(
     val auth: String? = null,
 
     @SerializedName("countryCode")
-    val countryCode: String? = null
-
-)
\ No newline at end of file
+    val countryCode: String? = null,
+
+    ) {
+
+    companion object {
+        fun fromJsonString(jsonString: String?): ChargeResponse {
```

Comment:
I dont see where we are using this. Is there a reason we have it? 😊

Reply:
`TransactionManager` 🤓

Reply:
sweet =)